controllers/controllers/cluster_controller.go

```diff
@@ -115,6 +133,31 @@ func (r *ClusterReconciler) reconcile(ctx context.Context, cluster *anywherev1.C
 	return reconcileResult.ToCtrlResult(), nil
 }
 
-func (r *ClusterReconciler) reconcileDelete(ctx context.Context, cluster *anywherev1.Cluster, log logr.Logger) (ctrl.Result, error) {
+func (r *ClusterReconciler) reconcileDelete(ctx context.Context, cluster *anywherev1.Cluster) (ctrl.Result, error) {
+	capiCluster := &clusterv1.Cluster{}
+	capiClusterName := types.NamespacedName{Namespace: "eksa-system", Name: cluster.Name}
+	r.log.Info("Deleting", "name", cluster.Name)
+	err := r.client.Get(ctx, capiClusterName, capiCluster)
+
+	switch {
+	case err == nil:
+		r.log.Info("Deleting CAPI cluster", "name", capiCluster.Name)
+		if err := r.client.Delete(ctx, capiCluster); err != nil {
+			r.log.Info("Error deleting CAPI cluster", "name", capiCluster.Name)
+			return ctrl.Result{}, err
+		}
+		return ctrl.Result{RequeueAfter: defaultRequeueTime}, nil
+	case !apierrors.IsNotFound(err):
```

Comment:
nit: I would move this down and put it under `default:`

Reply:
done